File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
#오븐 시계
h,m=map(int,input().split())
t=int(input())
result_m = m+t
if result_m>=60:
    h=h+result_m//60
    if h>=24:
        h=h-24
    result_m= result_m%60
print(h,result_m)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let h = Number(input[0].split(" ")[0]);
let m = Number(input[0].split(" ")[1]);
let time = Number(input[1]);

if (m + time >= 60) {
  h += Math.floor((m + time) / 60);
  m = (m + time) % 60;
} 
else {
  m += time;
}

if (h >= 24) {
  h -= 24;
}

console.log(h, m);
